index.html

```diff
@@ -260,6 +260,16 @@ <h2>The Citizenship Vocabulary</h2>
 A representative image of the individual.
             </td>
           </tr>
+          <tr>
+            <td>
+imageHash
```

Comment:
This is the wrong thing to do. We should be using hashlinks if we want to link to content-addressed content external to a Verifiable Credential:

https://tools.ietf.org/html/draft-sporny-hashlink-04

An alternative is to use a data URL (which is what we're doing for now):

https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs

Reply:
Manu, I have updated the branch with your corrections

Reply:
Hmm, what I'm saying is that imageHash is completely unnecessary in both scenarios, 1) Where you are embedding the image, or 2) When you are referring to the image using a hashlink. This is what a hashlink to the image could look like:

```
  "image": "hl:zQmWvQxTqbG2Z9HPJgG57jjwR154cKhbtJenbyYTWkjgF3e:zuh8iaLobXC8g9tfma1CSTtYBakXeSTkHrYA5hmD4F7dCLw8XYwZ1GWyJ3zwF"
```

or

```
  "image": "https://images.example.com/people/23849723947?hl=zQmWvQxTqbG2Z9HPJgG57jjwR154cKhbtJenbyYTWkjgF3e"
```